src/utils/env.ts

```diff
@@ -67,10 +67,19 @@ export const env = createEnv({
     RETRY_TX_ENABLED: boolSchema("true"),
     MAX_FEE_PER_GAS_FOR_RETRY: z.string().default("55000000000"),
     MAX_PRIORITY_FEE_PER_GAS_FOR_RETRY: z.string().default("55000000000"),
-    MAX_RETRIES_FOR_TX: z.coerce.number().default(3),
+    MAX_RETRIES_FOR_TX: z.coerce.number().default(15),
```

Comment:
3 was way too short, all my transactions were getting retried on mumbai polygon

Reply:
We should switch back to 3

Reply:
My bad, I wanted to update something else. Fixed now